reduced-actionbar: bag QoL

luskanek opened the pull request:
Hello shagu.

I added the option to replace a bag or pick it up without needing to disable the reduced actionbar option. To replace a bag, pick up a bag from one of your bags and click on the portrait of the desired bag that you want to replace. To pick up a bag, click on the portrait of the bag that you want to pick up. The bags need to be empty of course.

It's a simple addition but removes the hassle of needing to disable the reduced actionbar option just to manage your bags. The gif below is a bit wonky but it should suffice as a demonstration.

![peek](https://github.com/shagu/ShaguTweaks/assets/46632393/548db0dd-32ab-4a27-87bd-151284513e6b)

shagu commented:
Hello! I love that idea. 

Is this tested well already? Curious about the hardcoded `19` in there:
```lua
  local id = 19 + this:GetID()
```
I'm not really into the bag code anymore to know about that. 
I would love to have this merged, so If you think it's ready to go, please let me know :)

PS: please put this ontop:
```lua
local _G = ShaguTweaks.GetGlobalEnv()
```
and use `_G[]` instead of `getglobal()` to have everything same across all modules.

luskanek commented:
> https://github.com/shagu/ShaguTweaks/pull/68#issuecomment-1909804043

Updated the code.

While looking for the explanation as to why I used the hardcoded 19 (the parameters section of these two API function explains it: [PickupBagFromSlot](https://wowwiki-archive.fandom.com/wiki/API_PickupBagFromSlot), [PutItemInBag](https://wowwiki-archive.fandom.com/wiki/API_PutItemInBag)) I found out that [ContainerIDToInventoryID](https://wowwiki-archive.fandom.com/wiki/API_ContainerIDToInventoryID) exists, so I replaced the calculation with this function.

Tested this and found no issues.
